packages/build-utils/src/edge-function.ts

```diff
@@ -33,12 +33,18 @@ export class EdgeFunction {
    */
   envVarsInUse?: string[];
 
+  /**
+   * Extra binary files to be included in the edge function
+   */
+  blobs?: { name: string; path: string }[];
+
   constructor(params: Omit<EdgeFunction, 'type'>) {
     this.type = 'EdgeFunction';
     this.name = params.name;
     this.deploymentTarget = params.deploymentTarget;
     this.entrypoint = params.entrypoint;
     this.files = params.files;
     this.envVarsInUse = params.envVarsInUse;
+    this.blobs = params.blobs;
```

Comment:
Is this going to be confusing calling this `blob` because `files` can have `FileBlob` types? Seems like we should call this something else like `assets`?